### ISSUE TITLE
Unable to Open Multiple OniVim Windows on Ubuntu

### ISSUE THREAD

**Issue by Christopher-Hayes** [onivim/oni2#3633]
## Summary

When I have one OniVim window open, I am unable to open a second OniVim window / instance.

## Description

Different ways I've tried opening a second instance:

- Running the .AppImage with OniVim already open will just focus the OniVim that is currently open (Ubuntu will also send a "OniVim is ready" desktop notification)
- I have a `.desktop` file setup with OniVim, which allows me to add OniVim to my taskbar "favorites". It also means if I `right-click` on the app, it gives an "Open in a New Window" option. That will not open a new window; however.
- Using `CTRL` + `ENTER` when running the `.AppImage` from the shell is for running a new instance of an application. This didn't work either and only printed the result `[1]+   Done`.

**In conclusion, it seems like OniVim needs to specifically support multiple instances.**

## Use-Case

I would like to use OniVim on multiple projects.

## Specs

Version: `0.5.9-nightly` | `7c00d368`
OS: `Ubuntu 20.04`

**Comment by CrossR** [comment]
Hey!

We do actually support multiple windows, it was just the default behaviour was swapped to reuse the existing instance. That makes something like `oni2 file.txt` reuse it rather than open up a second window, which isn't usually desirable.

There is a `-n` /`--new` flag for the CLI that will always open a new window, but I don't know how well that will work for the desktop file case. I seem to think you can pass args in a desktop file, though it would be good for us to check/compare our behaviour to something like VSCode.

**Comment by Christopher-Hayes** [comment]
Ah, okay. I understand the use-case. So, VSCode and other applications like browsers if you open a file, it'll reuse the current window. Which OniVim seems to do similarly. The only trouble is, it's like OniVim specifically prevents a new window from being opened. The second and third options listed in the bug description are standard methods on Ubuntu of opening up a new window for application. OniVim seems to break away from that.

Thank you for the `-n` option, I'll start using that.
